charts/datadog/README.md

```diff
@@ -1,6 +1,6 @@
 # Datadog
 
-![Version: 2.26.5](https://img.shields.io/badge/Version-2.26.5-informational?style=flat-square) ![AppVersion: 7](https://img.shields.io/badge/AppVersion-7-informational?style=flat-square)
+![Version: 2.27.0](https://img.shields.io/badge/Version-2.25.0-informational?style=flat-square) ![AppVersion: 7](https://img.shields.io/badge/AppVersion-7-informational?style=flat-square)
```

Comment:
The link is wrong. It's pointing to 2.25.0.

Reply:
Thanks for pointing that out. Running `helm-docs.sh` was the solution.